pkg/bgpv1/types/bgp.go

```diff
@@ -114,6 +115,19 @@ type RoutePolicyConditions struct {
 	MatchFamilies []Family
 }
 
+// String() constructs a string identifier
+func (r RoutePolicyConditions) String() string {
```

Comment:
This is composing a hashable/comparable string identifier for the instance of `RoutePolicyConditions`.  Think map key.